Scenes/Main.tscn

```diff
@@ -94,11 +94,16 @@ scale = Vector2( 4, 4 )
 
 [node name="AnimationPlayer" type="AnimationPlayer" parent="Spaceship"]
 anims/spaceshipArrives = SubResource( 1 )
+<<<<<<< HEAD
```

Comment:
It seems that the merge was not resolved correctly. Did you ever had this problem? Do you know how to fix it?